server/tools/video_generators.py

```
@@ -0,0 +1,147 @@
+from abc import ABC, abstractmethod
+from typing import Tuple, Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)